### DISCUSSION TITLE
Any plans to make a conda package for scalene?

### DISCUSSION THREAD

**Discussion by robinfriedrich** [plasma-umass/scalene#386] [Q&A] [answered]
Would love that.

**Comment by emeryberger** [comment, marked as answer]
Yes! I just cleaned up [a pull request to add it as a conda recipe](https://github.com/conda-forge/staged-recipes/pull/18747).

**Reply by emeryberger** [reply]
If you are interested in making this happen, [please up-vote the Conda PR](https://github.com/conda-forge/staged-recipes/pull/18747#issuecomment-1207458259).

**Reply by emeryberger** [reply]
This is done; see https://anaconda.org/conda-forge/scalene.
